cmd/vega/nodewallet/import.go

```diff
@@ -60,6 +64,10 @@ func (opts *importCmd) Execute(_ []string) error {
 	log := logging.NewLoggerFromConfig(logging.NewDefaultConfig())
 	defer log.AtExit()
 
+	if (opts.EthereumClefAccount != "") != (opts.EthereumClefAddress != "") {
+		return ErrClefOptionMissing
+	}
```

Comment:
Just to confirm, you are expecting that both clef settings are either specified, or not at all right?

Reply:
Yup, thats the one.